src/cris-test-harness-cli/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cris-test-harness-cli",
+  "version": "0.3.0",
+  "description": "Package for running the cris-test-harness from the command line",
+  "main": "lib/main.js",
+  "types": "lib/main.d.ts",
+  "scripts": {
+    "build": "npx tsc"
+  },
+  "author": "@jolly59, @nhuntwalker",
+  "license": "ISC",
+  "dependencies": {
+    "colors": "^1.3.3",
+    "diff": "^4.0.1",
+    "microsoft-cognitiveservices-speech-sdk": "^1.5.0",
+    "tslib": "^1.10.0",
+    "typescript": "^3.5.2",
+    "word-error-rate": "0.0.7",
+    "yargs": "13.2.4"
+  },
+  "devDependencies": {
+    "@types/async": "^2.4.2",
+    "@types/diff": "^4.0.2",
+    "@types/jest": "^24.0.13",
+    "@types/node": "^12.0.0",
+    "@types/yargs": "^13.0.0",
+    "jest": "^24.8.0",
+    "ts-jest": "^24.0.2",
+    "ts-node": "^8.1.0"
+  },
+  "gitHead": "731c25486adc924a180e04f9dad46ee8bf4eb6f2"
```

Comment:
is this `githead` pointing at the current d-ser-t repo ?

Reply:
The `githead` I'm seeing in `package.json` now is pointing at the `nmhw/lerna` branch in this repository.